tests/integration/kubernetes/tests_common.sh

```diff
@@ -46,7 +46,7 @@ exec_host() {
 	command+='; echo -en \\n$?'
 	output="$(kubectl debug -qit "${node}" --image=alpine:latest -- chroot /host bash -c "${command}")"
 	kubectl get pods -o name | grep node-debugger | xargs kubectl delete > /dev/null
-	exit_code="$(echo "${output}" | tail -1)"
-	echo "$(echo "${output}" | head -n -1)"
+	exit_code="$(echo "${output}" | tail -1 | tr -d '\r')"
```

Comment:
What about adding some comment to clearly indicate this is an acceptable workaround until we know exactly what's happening here ?